templates/vars.yml.j2

```diff
@@ -140,9 +129,6 @@ matrix_prometheus_postgres_exporter_database_username: '{{PROMETHEUS_EXPORTER_PO
 matrix_prometheus_postgres_exporter_database_password: '{{PROMETHEUS_EXPORTER_POSTGRES_PASSWORD}}'
 {% endif %}
 
-matrix_postgres_process_extra_arguments: [
```

Comment:
We no longer need this then?

Reply:
Nop. I'd leave it for local testing and support in case of non-RDS use cases.

Reply:
Let's do that if it's helpful.